vars/perfRegressionParallelPipeline.groovy

```diff
@@ -127,7 +127,11 @@ def call(Map pipelineParams) {
                                                         rm -fv ./latest
 
                                                         export SCT_CLUSTER_BACKEND=${params.backend}
-                                                        export SCT_REGION_NAME=${pipelineParams.aws_region}
+                                                        if [[ ! -z "${aws_region}" ]]; then
```

Comment:
Please, use `-n` instead of `! -z`

And I'm little bit puzzled about where `aws_region` is defined.

Reply:
> Please, use `-n` instead of `! -z`
>

fixed

> 
> And I'm little bit puzzled about where `aws_region` is defined.
it is defined in [this line](https://github.com/scylladb/scylla-cluster-tests/blob/c0b07d9dab9db832353454de41c51f41bd702210/vars/perfRegressionParallelPipeline.groovy#L23)

Reply:
Should it be `params.aws_region` in that case?